parts/22-query.sh

```diff
@@ -2286,52 +2286,63 @@ query_group-gpu-time() { ##? [group]: Retrieve an approximation of the GPU time
 	EOF
 }
 
-query_monthly-users-registered(){ ## [year] [--by_group]: Number of users registered each month
+query_monthly-users-registered(){ ## [year] [YYYY-MM] [--by_group]: Number of users registered
```

Comment:
so this one is a bit odd for me, with year + yyyy-mm support. These arguments effectively act as filter, and I think we can simplify them, and make them match up a bit better with other commands like `query_monthly-job-runtimes`

```suggestion
query_monthly-users-registered(){ ## [--year=<YYYY>] [--month=<MM>] [--by_group]: Number of users registered
```

I think this could then be copied and pasted across the rest and we can remove the (lovely!!) use of regexes for matching strings and have a nicer more consistent UX.